### PR TITLE
PXC-4270 8.0.33-25 - release notes and release notes tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/release-notes/8.0.33-25.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by kamil-holubicki — COMMENTED** [review]
Hi @patrickbirch,
Looks fine, but in the meantime we decided we will have 3 more fixes:
https://jira.percona.com/browse/PXC-4258
https://jira.percona.com/browse/PXC-4268
https://jira.percona.com/browse/PXC-4278
Please add these references.

**Comment by patrickbirch** [comment]
Added the three tickets.

File: docs/release-notes/8.0.33-25.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Percona XtraDB Cluster 8.0.33-25 (2023-08-02)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.33-25 (2023-06-15) release notes](https://docs.percona.com/percona-server/8.0/release-notes/8.0.33-25.html).
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.33 and included in Percona Server for MySQL are the following:
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0.33-25.md]: included in Percona XtraDB Cluster

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**